Firestore/Example/Tests/API/FSTAPIHelpers.h

```diff
@@ -62,31 +32,35 @@ extern "C" {
 /** A convenience method for creating dummy singleton FIRFirestore for tests. */
 FIRFirestore *FSTTestFirestore();
 
-/** Creates a new GeoPoint from the latitude and longitude values */
-FIRGeoPoint *FSTTestGeoPoint(double latitude, double longitude);
-
 /** A convenience method for creating a doc snapshot for tests. */
 FIRDocumentSnapshot *FSTTestDocSnapshot(NSString *path,
                                         FSTTestSnapshotVersion version,
                                         NSDictionary<NSString *, id> *data,
                                         BOOL hasMutations,
                                         BOOL fromCache);
 
-/** A covenience method for creating a collection reference from a path string. */
+/** A convenience method for creating a collection reference from a path string. */
 FIRCollectionReference *FSTTestCollectionRef(NSString *path);
 
-/** A covenience method for creating a document reference from a path string. */
+/** A convenience method for creating a document reference from a path string. */
 FIRDocumentReference *FSTTestDocRef(NSString *path);
 
 /**
  * A convenience method for creating a particular query snapshot for tests.
- * This function allows user to pass in snapshot of the query in the past as well as new rows to be
- * added into the snapshot as for now. The current snapshot of the query consists both data.
+ *
+ * @param path To be used in constructing the query.
+ * @param oldDocs Provides data to construct the query snapshot in the past. It maps each key to a
+ * document. The key is the document's path relative to the query path.
```

Comment:
nit: "document's path" suggests this could be deeply nested under the query path, but it really must only be a single level deep. I would say:

Provides the prior set of documents in the QuerySnapshot. Each dictionary entry maps to a document, with the key being the document id, and the value being the document contents.